integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -313,8 +324,17 @@ def create_webhook(self, group: Group, events: list[str] | None) -> str | None:
         if group_id is None:
             logger.info(f"Group {group.attributes['full_path']} has no id. skipping...")
         else:
-            if self._does_webhook_exist_for_group(group):
+            hook = self._get_webhook_for_group(group)
```

Comment:
Can you add some tests? We need to make sure this functionality will not be broken in future releases